src/components/dapps/ModalCodeHash.vue

```diff
@@ -82,16 +82,64 @@
   </div>
 </template>
 <script lang="ts">
-import { defineComponent } from 'vue';
+import { defineComponent, ref, computed } from 'vue';
+import { useStore } from 'vuex';
+import { ActionTypes } from '@/store/action-types';
 
 export default defineComponent({
+  props: {
+    address: {
+      type: String,
+      required: true,
+    },
+  },
   setup(props, { emit }) {
     const closeModal = () => {
       emit('update:is-open', false);
     };
 
+    const store = useStore();
+    const api = computed(() => store.getters.api);
```

Comment:
I noticed that the api instance from the global store and the API instance from the injected context is used throughout the codebase. Please remove the API state in the global store and change all existing references to use the `useApi` method instead. This is hard to debug.

https://github.com/PlasmNetwork/plasm-apps-vue/blob/9d0538083fe77423370fdec5351de34e010047c4/src/hooks/useApi.ts#L6

https://github.com/PlasmNetwork/plasm-apps-vue/blob/732acb169c0ca8d4d39a4b05d74d32bd5476c637/src/store/state.ts#L21

Reply:
it's similar reason what I chose. https://github.com/PlasmNetwork/plasm-apps-vue/pull/24#discussion_r643277348 
if `useApi` is fixed, it would be resolved. will fix at first then